src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -204,6 +204,8 @@ public function toolbarAction(Request $request, $token)
             $url = $this->generator->generate('_profiler', array('token' => $token));
         } catch (\Exception $e) {
             // the profiler is not enabled
+        } catch (\Throwable $e) {
+            // the profiler is not enabled
```

Comment:
> Yeah, but the generator can be user-provided... so what then?

if they provide something not respecting the interface, letting things bubble higher is OK